filament/src/details/Material.cpp

```diff
@@ -1140,13 +1140,30 @@ void FMaterial::precacheDepthVariants(FEngine& engine) {
 void FMaterial::processDescriptorSets(FEngine& engine, MaterialParser const* const parser) {
     UTILS_UNUSED_IN_RELEASE bool success;
 
+
     success = parser->getDescriptorBindings(&mProgramDescriptorBindings);
     assert_invariant(success);
 
     std::array<backend::DescriptorSetLayout, 2> descriptorSetLayout;
     success = parser->getDescriptorSetLayout(&descriptorSetLayout);
     assert_invariant(success);
 
+    // Setup Labels for debugging
+    utils::CString namestr;
+    utils::StaticString perViewStr = "_perView";
+    utils::StaticString singleStr = "_single";
+
+    parser->getName(&namestr);
```

Comment:
You should be able to use `mName` here, which is initialized inside the `FMaterial` constructor.